client/assets/content_bundles/symptom_poster.en.yaml

```diff
@@ -1,39 +0,0 @@
-schema_version: 1
```

Comment:
Can't delete since old apps rely on this url/file specifically.

Reply:
Good point. What's the way to add a comment so we delete it later? Or maybe we should just track the delete with a GitHub issue.

Reply:
I have restored it.  We really only need this file on the server now though... We should have a place to archive them.